core/templates/components/graph-services/graph-layout.service.ts

```diff
@@ -73,11 +73,11 @@ export class StateGraphLayoutService {
 
   // The last result of a call to computeLayout(). Used for determining the
   // order in which to specify states in rules.
-  lastComputedArrangement: NodeDataDict = null;
+  lastComputedArrangement: NodeDataDict | null = null;
```

Comment:
Change the comment to include what does `null` mean.

Reply:
Not done.

Reply:
check now, actually, I forgot to commit that

Reply:
Looks good.